internal/provider/data_source_catalystcenter_fabric_site.go

```diff
@@ -109,31 +99,9 @@ func (d *FabricSiteDataSource) Read(ctx context.Context, req datasource.ReadRequ
 	}
 
 	tflog.Debug(ctx, fmt.Sprintf("%s: Beginning Read", config.Id.String()))
```

Comment:
What specifically did you change in the "Read" method? And if we change something manually we need to remove the section markers.

Reply:
I added this query param: 	params += "?id=" + url.QueryEscape(config.Id.ValueString()), but for some reason i missed markers removal, I pushed new commit with correct Read function